dev/firely/docker-compose.mssqlserver.yaml

```
@@ -0,0 +1,31 @@
+version: '3'
+
+services:
+
+  firely:
+    depends_on:
+      - db
+    environment:
+      # https://docs.fire.ly/projects/Firely-Server/en/latest/setting_up_firely_server/deployment/docker.html#running-docker-with-a-sql-server-container
+      VONK_Repository: SQL
+      VONK_SqlDbOptions:AutoUpdateConnectionString: Initial Catalog=VonkData;Data
+        Source=db,1433;User ID=sa;Password=SQLServerStrong(!)Password*
+      VONK_SqlDbOptions:AutoUpdateDatabase: "true"
+      VONK_SqlDbOptions:ConnectionString: Initial Catalog=VonkData;Data Source=db,1433;User
+        ID=sa;Password=SQLServerStrong(!)Password*
+      VONK_SqlDbOptions:SchemaName: vonk
+    networks:
+      - internal
+      - ingress
+
+  db:
+    image: mcr.microsoft.com/mssql/server:${DB_IMAGE_TAG:-2022-latest}
+    environment:
+      ACCEPT_EULA: "Y"
+      SA_PASSWORD: SQLServerStrong(!)Password*
+    networks:
+      - internal
```

You'll want to keep the MSSQL DB files on a separate volume, otherwise the data will be deleted when the container is deleted.

[Configure and customize SQL Server Docker containers:Use data volume containers](https://learn.microsoft.com/en-us/sql/linux/sql-server-linux-docker-container-configure?view=sql-server-ver16#use-data-volume-containers)

```suggestion
      - internal
    volumes:
      - db-data:/var/opt/mssql
  volumes:
      db-data: {}
```